src/provider/cloud-element-templates/ElementTemplates.js

```diff
@@ -62,6 +69,55 @@ export default class ElementTemplates extends DefaultElementTemplates {
 
     return context.element;
   }
+
+  /**
+   * Unlink template from a given element.
+   *
+   * @param {djs.model.Base} element
+   */
+  unlinkTemplate(element) {
+    const modeling = this._modeling,
+          eventBus = this._eventBus;
+
+    eventBus.fire('elementTemplates.unlink', { element });
+
+    // remove template attributes
+    modeling.updateProperties(element, {
+      'zeebe:modelerTemplate': null,
+      'zeebe:modelerTemplateVersion': null,
+      'zeebe:modelerTemplateIcon': null
+    });
+  }
+
+  /**
+   * Update template to latest version.
+   *
+   * @param {djs.model.Base} element
+   * @param {ElementTemplate} newTemplate
+   * @returns {djs.model.Base} the updated element
+   */
+  updateTemplate(element, newTemplate) {
```

Comment:
The reason we have this (over `apply`) is that we want to tell appart apply and update, right? Fine with me.

Reply:
Alternatively we could emit an event in `apply` that allows us to distinguish `apply` and update. Then we have one less API to maintain.

Reply:
So keeping only `applyTemplate` in the API and fire `.update`/`.apply` based on if the templates are the same or not (minus the version)?

Reply:
Yea. Or just fire apply (because from the properties panel point of view there is only _apply_, but provide enough context so that integrators can tell apart apply and update.

Reply:
I decided to merge the API function but separate the events. We already assume a `update` event with tracking so let's just keep it

Reply:
I like that reasoning. :)